docs/tutorials/create-pages.md

```diff
@@ -0,0 +1,178 @@
+<h1>Tutorial: Creating new pages</h1>
+
+This tutorial will describe how to create new pages and add them to your main menu. It will also cover some steps needed for multilingual implementations.
+
+## Topics covered
+
+* Jekyll pages
+* Markdown
+* Multilingual pages
+* Menu configuration
+
+## Level of difficulty
+
+This tutorial is mostly about content, but does involve the editing of some configuration files. Anyone can follow the steps to complete the tutorial, however you will be using the YAML syntax involved in most Open SDG configurations. YAML requires particular indentation and spacing, so some attention to detail will be needed.
```

Comment:
Would it be a good idea to link to some examples of yaml syntax, [like this](https://blog.stackpath.com/yaml/)? Or would that be over complicating things?